src/e2eTest/kotlin/io/github/gradlenexus/publishplugin/e2e/NexusPublishE2ETests.kt

```diff
@@ -37,6 +37,7 @@ class NexusPublishE2ETests : BaseGradleTest() {
         //when
         val result = run(
             "publishToSonatype",
+            "findSonatypeStagingRepository",
```

Comment:
I could probably move that to a different test if you think it is needed, however, I would love to hear some feedback first

Reply:
I've created a separate "two stage" release testin E2ETests